### PR TITLE
Add 2018 premium sponsors

### PR DESCRIPTION
**Pull request by skamoen**

### CODE REVIEW

**Comment by skamoen** [comment]
image

File: src/lancie-sponsors/lancie-sponsors.html [hunk 1]
```diff
@@ -48,6 +48,13 @@ <h3>Presented by</h3>
         <a href="https://ch.tudelft.nl" target="_blank" rel="noopener noreferrer"><img src="images-optimized/lancie/logo_CH.png"></a>
         <a href="https://www.tudelft.nl" target="_blank" rel="noopener noreferrer"><img src="images-optimized/logos/logo_SC.png"></a>
       </div>
+      <div class="layout vertical flex-4">
+        <h3>Premium sponsors</h3>
+        <div class="layout horizontal wrap premium-sponsors">
```
Comment (elarb) [inline src/lancie-sponsors/lancie-sponsors.html]: Why do we have a separate class for the same style?

**Comment by skamoen** [comment]
We've been trying to get this properly styled for 2 years. This PR is about content,which is at this moment way more important than pretty html

**Comment by skamoen** [comment]
That fixes the wrapping. Other fixes will come when we reimplement this again with ES6 and dynamic loading

**Pull request merged**